app/src/main/java/com/springboot/webpush/service/NotificationService.java

```diff
@@ -19,24 +19,41 @@
 import nl.martijndwars.webpush.Notification;
 import nl.martijndwars.webpush.PushService;
 import nl.martijndwars.webpush.Urgency;
+import java.security.KeyPair;
+import java.security.PublicKey;
 
 @Service
 public class NotificationService {
 	private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
 	private final SubscriptionService subscriptionService;
 	private final ObjectMapper objectMapper;
-	private final PushService pushService;
+	private PushService pushService;
+	private final KeyPair keyPair;
 
 	public NotificationService(SubscriptionService subscriptionService, ObjectMapper objectMapper,
 			final KeyPairService keyPairService) {
 		super();
 		this.subscriptionService = subscriptionService;
 		this.objectMapper = objectMapper;
+		this.keyPair = keyPairService.generate().getKeyPair();
+
+		/*
+		 * this.pushService = new PushService(this.keyPair) //
+		 * .setSubject("mailto:admin@domain.com");
+		 */
+		try {
+			this.pushService = new PushService(
+					"BJ3kKgvZR34Wt-dJMG9L7Lh03ISRlWueIgg-Vp5V5oYsDPr47_szfNsxxLQrghPtuN3XaI6A3FVZVZuVa7LZOFY",
```

Comment:
First one is the hard-coded vapid public key, second is the hard-coded vapid private key. Should try to make this work from the keypair.